install.sh

```diff
@@ -47,12 +47,14 @@ execute() {
   # hash_sha256_verify "${tmpdir}/${TARBALL}" "${tmpdir}/${CHECKSUM}"
   srcdir="${tmpdir}"
   (cd "${tmpdir}" && unzip -o "${TARBALL}")
-  install -d -m 775 "${BINDIR}" 2> /dev/null || install -d "${BINDIR}"
+  log_debug "setting up bindir: $BINDIR"
+  mkdir -p $BINDIR
   for binexe in "fossa" ; do
     if [ "$OS" = "windows" ]; then
       binexe="${binexe}.exe"
     fi
-    install "${srcdir}/${binexe}" "${BINDIR}/" 2> /dev/null || sudo install "${srcdir}/${binexe}" "${BINDIR}/"
+    log_debug "installing binary: $binexe"
+    install "${srcdir}/${binexe}" "${BINDIR}/"
```

Comment:
Is there a reason we're removing the `sudo` option?

Reply:
No, that was for testing, I will re-add it.